en/docs/reference/mediators/callout-mediator.md

```diff
@@ -60,7 +60,7 @@ The parameters available for configuring the Callout mediator are as follows.
 <li><strong>URL</strong> : Select <strong>URL</strong> if you want to call the external service by specifying its URL in the Call mediator configuration.</li>
 <li><strong>Address Endpoint</strong>: Select <strong>Address Endpoint</strong> if you want to call the external service via an <b>Endpoint</b>, which is already saved in the <b>Registry</b>. This option should be selected if you want to make use of the WSO2 functionality related to endpoints such as format conversions, security etc. Note that only Leaf endpoint types (i.e. <code>                Address               </code>, <code>WSDL</code>, <code>Default</code> and <code>Http</code>) are supported for the Callout mediator.</li>
 </ul>
-<p>If neither a URL or an address endpoint is specified, the <code>To</code> header on the request is used as the target endpoint.</p>
+<p>If neither a URL nor an address endpoint is specified, the <code>To</code> header on the request is used as the target endpoint.</p>
 </div></td>
```

Comment:
The SOAP action which should be appended to the service call
->
The SOAP action should be appended to the service call